thumbor_shortener/handlers/shortener.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from thumbor_shortener.shortener import Shortener
+from thumbor.handlers.imaging import ImagingHandler
+from thumbor.url import Url
+
+class UrlShortenerHandler(ImagingHandler):
+
+    def get(self, **kwargs):
+
+        # Get the url from the shortener and parse the values.
+        url = Shortener.get(self.request.path)
```

Comment:
Yep. We could imagine a system of extensions that would be able to edit the context once instantiated, but also add handlers, loaders, storages, and so on.